### PR TITLE
Feature/highlight new comments

### PR DESCRIPTION
**Pull request by victorgcramos**

### CODE REVIEW

**Comment by victorgcramos** [comment]
Fixes #732

**Comment by victorgcramos** [comment]
https://github.com/decred/snew-classic-ui/pull/24

File: src/components/snew/ThingComment.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ class ThingComment extends React.PureComponent {
       ...props
     } = this.props;
     const isProposalAbandoned = proposalStatus === PROPOSAL_STATUS_ABANDONED;
+    console.log(lastVisit);
```
Comment (tiagoalvesdulce) [inline src/components/snew/ThingComment.js]: You left a log here.

**Comment by victorgcramos** [comment]
This happens because the accesstime is set only when the proposal is fetched. This was discussed on Slack and I was told that this was the best way to deal with the situation, since I was not allowed to set the accesstime on a new request.

**Comment by tiagoalvesdulce** [comment]
LGTM

**Pull request merged**